modules/about-rhde.adoc

```diff
@@ -53,6 +55,10 @@ The two products of {product-title} work together as a single solution for devic
 ^|None
 |===
 
+When {ansible} is used to manage {product-title} deployments, there is no hard dependency. Using the latest {ansible} is recommended, but any supported version is possible. See the https://access.redhat.com/support/policy/updates/ansible-automation-platform[Red Hat Ansible Automation Platform Life Cycle] page for details.
```

Comment:
```suggestion
When {ansible} is used to manage {product-title} deployments, there is no hard dependency. Using the latest {ansible} is recommended, but you can use any supported version. See the https://access.redhat.com/support/policy/updates/ansible-automation-platform[Red Hat Ansible Automation Platform Life Cycle] page for details.
```

Reply:
+1 
"hard dependency" sound a bit vague and it might help to expand or clarify its meaning here